src/toil/provisioners/abstractProvisioner.py

```diff
@@ -184,6 +187,25 @@ def readClusterSettings(self):
         """
         raise NotImplementedError
 
+    def _writeGlobalFile(self, key: str, contents: bytes) -> str:
+        """
+        Write a file to a physical storage system that is accessible to the
+        leader and all nodes during the life of the cluster.
+
+        Not to be confused with the `writeGlobalFile()` method in the file
```

Comment:
Wouldn't it be easier to avoid confusion by naming this concept something other than a "global file"? Why did you pick that name?

Reply:
Yes, I agree. The only reason for the name is that I couldn’t think of anything else. Any suggestions for a better name?